src/components/SignUpForm/index.js

```diff
@@ -25,6 +25,12 @@ const FIELDS = {
   gender: 'gender',
 };
 
+const {
+  SIGN_UP: { name, passwordPlaceholder, passwordConfirmation, gender, button },
+  COMMON: { email, password },
+  GENDER: { options, placeholder },
+} = strings;
```

Comment:
I'm not sure I would deconstruct this. I like to know this is a copy by looking at the variable name.